### PR TITLE
Weekly overview: fix width of table

### LINKED ISSUE #4429: Missing horizontal bar
**Describe the bug** / **To Reproduce**
The icon "View" on the homepage -> "Week's events" shows a table that doesn't fit in a small screen, and there is no horizontal navigation bar to see the information of the last days. 
This also impact other pages where the content is bigger than the actual browser windows size (big table like registrants list).

**Expected behavior**
An horizontal bar should appear to let horizontal scrolling

weeks-events

**Additional context**
Change CSS to:

body {
    overflow-x: visible;

### PR DESCRIPTION
**Pull request by Leats**
Closes #4429.

Width of table now adjusts to screen size. Also takes care of long locations (like links) breaking the layout of the page.

### CODE REVIEW

File: indico/web/client/styles/modules/_category.scss [hunk 1]
```diff
@@ -445,135 +447,143 @@ ul.category-resource-qtip {
...
 
-        th {
-            background-color: #444;
-            color: #fff;
-            text-align: center;
-            font-weight: normal;
-            white-space: nowrap;
-            padding: 1px 15px;
+            th {
+                background-color: #444;
```
Comment (ParthS007) [inline indico/web/client/styles/modules/_category.scss:469]: Is the color not available in `pallate` of colors?

File: indico/web/client/styles/modules/_category.scss [hunk 2]
```diff
@@ -445,135 +447,143 @@ ul.category-resource-qtip {
...
-    }
+        td.content-info {
+            padding-left: 20px;
 
-    td.content-info {
-        padding-left: 20px;
+            table.timetable-entries {
+                tr {
+                    &.contribution {
+                        background-color: #deebf8;
```
Comment (ParthS007) [inline indico/web/client/styles/modules/_category.scss:552]: same

File: indico/web/client/styles/modules/_category.scss [hunk 3]
```diff
@@ -445,135 +447,143 @@ ul.category-resource-qtip {
...
-                }
+                    &.session {
+                        background-color: #fff1d5;
+                    }
 
-                &.session {
-                    background-color: #fff1d5;
-                }
+                    &.break {
+                        background-color: #e0ecd2;
```
Comment (ParthS007) [inline indico/web/client/styles/modules/_category.scss:560]: same

File: indico/web/client/styles/modules/_category.scss (hunk 1 above)
Comment (Leats) [reply indico/web/client/styles/modules/_category.scss:469]: There is no `#444`, and also the only thing I changed there was the indentation :) same for all the other ones

**Pull request merged**